examples/aiohttp/dogs/README.md

```diff
@@ -0,0 +1,8 @@
+# Tatiflette AIOHTTP - WebSockets subscriptions
+
+This is a fully Dockerised example.
```

Comment:
Can you add that docker and docker-compose are mandatory deps for this example to work ? (Add the version you validate the example with for both of them)